package.json

```diff
@@ -20,13 +18,26 @@
   "scripts": {
     "build": "rm -rf lib && babel src --out-dir lib --ignore *.spec.js",
     "test": "npm run test:lint && npm run test:unit && npm run test:integration",
-    "test:lint": "eslint .",
+    "test:lint": "standard | snazzy",
+    "test:lint:fix": "standard --fix",
```

Comment:
They are internal scripts for the Sagui project itself, and therefore are not exposed to a project using Sagui.